crates/jit/src/profiling/vtune_linux.rs

```diff
@@ -112,6 +112,9 @@ impl ProfilingAgent for VTuneAgent {
     fn module_load(&self, module: &CompiledModule, dbg_image: Option<&[u8]>) {
         self.state.lock().unwrap().module_load(module, dbg_image);
     }
+    fn trampoline_load(&self, _file: &object::File<'_>) {
+        // TODO: needs an implementation
```

Comment:
Heads up @jlb6740, since you're the original author of the VTune support: generated trampolines for calling into Wasm weren't previously being registered for profilers (see the top-level PR description) and there are some updates needed for including them in the VTune support. See the jitdump support in this PR for an idea of the kind of thing needed. I don't know exactly what is needed for VTune.